cpp/ps0000/ps0000_slon.cpp

```diff
@@ -0,0 +1,8 @@
+#include <iostream> // 1/2
+
+int main(void) // 1/1
+{
+  std::cout << "Hello, World!\n"; // 3/4 
```

Comment:
- [ ] used `<<` for outputting instead of `>>`
- [x] used `\n` or `std::endl` in the right way
- [x] avoided syntax error by typing `;` at the end of the line
- [x] output is as required `Hello, World!` in a single line